Userland/Libraries/LibWeb/DOM/CharacterData.cpp

```diff
@@ -72,6 +72,9 @@ WebIDL::ExceptionOr<void> CharacterData::replace_data(size_t offset, size_t coun
     Utf16View utf16_view { utf16_data };
     auto length = utf16_view.length_in_code_units();
 
+    auto inserted_data_utf16 = MUST(AK::utf8_to_utf16(data));
+    auto inserted_data_length_in_utf16_code_units = Utf16View { inserted_data_utf16 }.length_in_code_units();
```

Comment:
This can just be: `auto inserted_data_length_in_utf16_code_units  = AK::utf16_code_unit_length_from_utf8(data);`